test-utils.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+COMMAND="$1" # setup | setup-fork | update
```

Comment:
nitpick 😄 i think the comment here should be `# setup | setup-dev | update | remove` ?

Reply:
fixed 🫡